s3_test.go

```diff
@@ -87,9 +85,10 @@ func TestExportImport(t *testing.T) {
 	}
 	defer os.RemoveAll("./test_data")
 
-	if s, err = testInit(); err != nil {
-		t.Fatal(err)
-	}
+	//	if s, err = testInit(); err != nil {
```

Comment:
If we don't need this anymore, can we remove?

Reply:
decide to keep it for cleanup